packages/cloud-cognitive/src/components/NonLinearReading/_non-linear-reading.scss

```diff
@@ -0,0 +1,156 @@
+//
+// Copyright IBM Corp. 2023, 2023
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+// Standard imports.
+@import '../../global/styles/project-settings';
+
+@mixin term-closed-defaults {
+  // stylelint-disable-next-line carbon/layout-token-use
+  padding: 1px $spacing-03 1px $spacing-03;
```

Comment:
We try to stick with spacing tokens or at least rems where we can. Is the 1px just used to achieve a 20px height?

Reply:
Yes, the `1px` is to set the height.

Same problem as detailed below: using `rem()` and `carbon--rem()` throw linting errors.